src/content/docs/en/guides/assets.mdx

```diff
@@ -145,19 +145,57 @@ To avoid errors in your project, complete the following steps:
 
 ### Update content collections schemas
 
-A new `image` helper for content collections can validate the metadata of images located in the `src/assets/` folder, such as author photos or blog cover images.
+You can now declare images in your frontmatter as their paths relative to the `src/assets/` folder. The image will be imported and transformed into metadata that matches the signature of imported images, allowing you to pass its properties directly to the `<Image />` component or the `getImage()` function.
 
-The image metadata will be transformed to match the signature of imported images, and therefore can be passed directly to the `<Image />` component or the `getImage()` function.
+A new `image` helper for content collections lets you validate the image metadata using Zod.
```

Comment:
I'm tempted to put this phrase, and the example of the schema, below the example for basic usage that's described above.

Right now this feels like:
- paragraph explaining how to use it with frontmatter
- note explaining the schema helper
- example in a schema
- example of basic usage

So maybe instead??

- paragraph explaining how to use it with frontmatter
- example of basic usage
-  note explaining the schema helper
- example in a schema

Reply:
Reorganized!